### PR TITLE
Support bitmap(binary) type in hive to import doris

### PR DESCRIPTION
**Pull request by rufeng97**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [x] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java [hunk 1]
```diff
@@ -71,14 +71,25 @@ public static ColumnParser create(EtlJobConfig.EtlColumn etlColumn) throws Spark
...
+    }
 }
 
 class TinyIntParser extends ColumnParser {
     @Override
-    public boolean parse(String value) {
+    public boolean parse(Object value) {
         try {
-            Byte.parseByte(value);
+            Byte.parseByte(value.toString());
```
Comment (wangbo) [inline fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java:92]: If value is null, then NPE happens here.

Comment (rufeng97) [reply fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java:92]: NPE will also occur in the previous code, and NPE has been try catch.

Comment (spaces-X) [reply fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java:92]: NumberFormatException is included but NPE will not catched in this situation.

File: fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java [hunk 2]
```diff
@@ -90,6 +91,8 @@ public static Class getClassFromColumn(EtlJobConfig.EtlColumn column) throws Spa
                 return Double.class;
             case "DATE":
                 return Date.class;
+            case "BINARY":
+                return BitMap.class;
```
Comment (wangbo) [inline fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java:95]: What does BitMap.class used for?

**Comment by wangbo** [comment]
Please add ```User Manual``` for this feature

File: fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java (hunk 2 above)
Comment (rufeng97) [reply fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java:95]: I wrote it by mistake before, you can delete it.

**Comment by wangbo** [comment]
This pr would be closed  temporarily and will be re-pull request after internal test

**Pull request closed without merging**